setup.py

```diff
@@ -23,6 +23,8 @@
         'clickhouse': ['clickhouse-driver>=0.2.0'],
         'redis': ['python-redis>=0.2.1', 'redis>=2.10.6'],
         'mysql': ['PyMySQL>=0.9.2'],
+        # higher versions are not python3.6 compatible
+        'rabbitmq': ['aio-pika==7.2.0'],
```

Comment:
see https://github.com/yandex/yandex-taxi-testsuite/blob/develop/setup.py#L16 for example how to provide per-python dependency